docs/source/tutorials/benchmarking/bm_sagemaker.rst

```diff
@@ -108,6 +108,9 @@ used with ``hpo_main.py``. For the example above:
 The warm pool feature is most useful with multi-fidelity HPO methods (such as
 ``ASHA`` and ``MOBSTER`` in our example). Some points you should be aware of:
 
+* When using SageMaker managed warm pools with the SageMaker back-end, it is
```

Comment:
backend

Reply:
https://en.wikipedia.org/wiki/Frontend_and_backend

Seems like you are right, most of the time.